### ISSUE TITLE
General diskuccion

### ISSUE THREAD

**Issue by desputinski** [clementine-player/Clementine#6040]
As I am not willing to get an Google-account just to join the discussion (don't feed the google!), I would like to pose the question here:
I am wondering if Clementine is an active project. I can see some new dev-versions here:
https://builds.clementine-player.org/

But all in all the development has slowed down quite a bit as it seems. Last release from 2016! Does anyone of you have more details on the future of the Clementine-Project?

**Comment by gariasf** [comment]
I've been wondering the same for a long while. There's a 1.3.2 branch on this repo too, but nothing seems to be coming out.

**Comment by desputinski** [comment]
Hm, maybe the developers could say something regarding the future of Clementine?
it seems to be "David Sansome, John Maguire and Arnaud Bienner"

**Comment by vazhnov** [comment]
Similar issue: #6009.

**Comment by gariasf** [comment]
To be honest, there was a considerable gap between the 2013 release and the 2016 one, so perhaps the project is considered as kinda finished and every now and then they update it to fix new issues.

**Comment by TheBlinded** [comment]
After all this time without replies I deduce that.. Clementine-Project is dead....

**Comment by hatstand** [comment]
It's just resting

**Comment by ArnaudBienner** [comment]
I must confess that I have been inactive on this project for a quite a while (but I'm not the main contributor).
Because I'm working on other projects and simply have less time. And also because (as @gariasf noted) Clementine personally fits all my needs and I feel like this project is mature now.
I understand some people might want an official release including latest fixes (even though, as noted, dev builds are available), but releasing a new version is very time consuming. To actually do, and once it is released if some regressions are reported by users.

**Comment by a17r** [comment]
I don't think a new Clementine release is about getting fixes out. At this point a Qt4-based release would be largely irrelevant as far as distributions are concerned - those who do not care enough to make a snapshot based off qt5-branch will perhaps drop Clementine from their package repository...

**Comment by a17r** [comment]
No, Qt4 to Qt5 is a straightforward port for the most part.

**Comment by SaiyanRiku** [comment]
Do you have any plan for the pending pull request? @hatstand @ArnaudBienner 

I think that some features or bugs fix should be considered to be merged in the master branch.

Then if we have daily builds, it could be enough to get a version with the bugs fix. But i see that Windows build don't seems to work anymore, and there is no Debian stretch or buster builds.

Clementine is close to be perfect for me, i would like to contribute more to add some minor features i'm missing, but only if we have chance to get it integrated in the master.

**Comment by desputinski** [comment]
What is progress doing in connection to porting clementine to QT5? Anything new?

**Comment by AnthonyWC** [comment]
Maybe whoever want to take over should fork this project at this point before it completely dies off.  Interests still seem high yet the original folks are no longer maintaining this project.

**Comment by desputinski** [comment]
I would donate some money if somebody would continue the project. Is there a website where one can post a project like this so more people could donate to make it survive?

**Comment by SaiyanRiku** [comment]
It seems that there is regularly some commit in the master branch, with some active developers. I would like to contact the team to know how to help the project, (i'm c++/Qt developer), but forking the project is too much work just for me.

**Comment by desputinski** [comment]
Ok, that would be great, if you could contact them directly. I tried to contact the developer David Sansome some time ago but didn't get an answer. He implemented the moodbars into the program and was one of the main developers.

**Comment by juhele** [comment]
Hi,
just found out there is a Clementine fork called Strawberry Music Player:
https://github.com/jonaski/strawberry

I did not try it yet but looks similar and they say "t's based on a heavily modified version of Clementine created in 2012-2013". Latest updates are few days old.

**Comment by desputinski** [comment]
Hm, it looks like clementine. But for me it is only be an alternative, if it has moodbars integrated...

**Comment by Sal1950** [comment]
Strawberry was an interesting idea formed largely due to extremely arrogant posting by contributor hatstand that the Clementine developers feel "I don't think we care about "audiophile" things" in the issue thread started over the removing of the alsa output card options 
https://github.com/clementine-player/Clementine/issues/5344
Unfortunately Strawberry has been stripped of so many functions and features and is very limited in it's value.  Like desputinski I would gladly contribute some funds to Clementine's development I it's developers would consider to once again include the code that alowed us to direct bit perfect streams to out DAC's.  We don't pay for high resolution recordings and expensive USB DAC's only to have the stream resampled into something much less.

**Comment by Sal1950** [comment]
I need to update my above post.
Strawberry has continued to develop and add new features.
It is now the media player to have if you value High Fidelity music playback,
Thanks Jonas and all the rest of the Strawberry dev team!

**Comment by AnthonyWC** [comment]
@Sal1950 The playbook quality from Strawberry is amazing!

**Comment by Sal1950** [comment]
> @Sal1950 The playbook quality from Strawberry is amazing!

Yes it is, it is even passing my 5.1 24/96 multichannel files to my Marantz Pre/Pro in bit perfect condition. The backend configuration options allow the player to completely bypass Pulse and deliver the data stream directly to Alsa.  I love this player.  Shame about Clementine. :(

**Comment by JulianVolodia** [comment]
Assuming @Sal1950 you moved to Strawberry, am I right?

**Comment by Sal1950** [comment]
Yes, @JulianVolodia  that is correct.
Too bad the Clementine dev's have failed to keep up with the market needs.  :(
It was a great player.

**Comment by JulianVolodia** [comment]
Thanks for info. How you compare current player to Clementine?

**Comment by Sal1950** [comment]
Strawberry is better in most ways, specially for a audiophile. It handles direct data connection to alsa and can also handle 5.1 multichannel files. Only thing I really miss from Clementine is the remote android app.

**Comment by JulianVolodia** [comment]
Thanks @Sal1950 . Hi @jonaski - maybe you want to know that.

**Comment by Sal1950** [comment]
I see the thread on the alsa configuration tools has been locked due to someone using bad language in his approach to the dev's and the discussion. That's unfortunate as that type of thing never adds anything of value to a debate. 
But in any case I believe we can all consider Clementine to be closed issue itself. It's lack of any real maintenance in years along with some of the dev's attitude towards the Hi Fi community and it's user base in general has forced a very large section of us to move elsewhere.
I truly regret to see what at one time was a quality open source project die like this, it was my go-to media player for years. During my many years as a main player at PCLinuxOS we kept Clementine as the default player in our distribution. It's like losing an old friend.
RIP  Clementine.

**Comment by mashiro-tan** [comment]
Just wrote a post to that [alsa](https://github.com/clementine-player/Clementine/issues/5344) thread and found that thread was locked, huh. Whatever, i'll just post it here.

I'll just add my case in the bucket. I'm a meloman and i loved the amarok 1.4 appearance in the times when i've used KDE3. Now, after setting up the PA-less Devuan setup with LXQt desktop, i've decided to install its successor, Clementine. Decision was simple: small dep count, native interface (not GTK), known behaviour. 
Result? No sound. Absolutly. 
So i found this large thread, read the developer's policy and switched to the Strawberry. There is no any sense to use a music player which didn't produce sound when the core sound pipeline is good and working.
I prefer convenience before quality, but PA don't have not first, not second - i was tired from the PA's poor out-of-the-box sound quality, noticeable soundtrack delays in the movies, bugs and from manually setting up all this mess. Just nope. 

PS: from the user's point of view Clementine along with the DeadbeeF are the main close alternatives to the Windows' foobar2000, the common player in the hi-fi audio setups. And that's somewhat ironic, player that was forked due to broken reliance of experienced users now was forked again for the same reason.

**Comment by fitojb** [comment]
Wow, this thread has become a FUD fest. I recommend to lock it and delete it.

**Comment by Sal1950** [comment]
> Wow, this thread has become a FUD fest. I recommend to lock it and delete it.

Sorry but disagree on deletion. This thread holds valuable information for those searching for it on Clementine

**Comment by tn5421** [comment]
> Wow, this thread has become a FUD fest. I recommend to lock it and delete it.

"This thread shows just how shitty and arrogant my friends are, I recommend to lock and delete it"
